modules/shared-memory.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-ephemeral-storage.adoc
+
+[id="shared-memory_{context}"]
+= Shared Memory
+
+
+Shared memory is one of the fastest IPC (Inter-process communication) mechanisms in Linux. There are two types of shared memory objects: System V and POSIX.
+Since the containers in a pod share the IPC namespace of the pod infrastructure container, they can automatically share the System V shared memory objects.
+
+A POSIX (The Portable Operating System Interface) shared memory object is a memory-mapped file, which requires a tmpfs filesystem mounted at *_/dev/shm_*.
```

Comment:
You probably don't need the _The_ where you expanded on the acronym: A POSIX (Portable Operating System Interface)

Reply:
Addressed.